### ISSUE TITLE
Image not showing in release mode with Flutter Flavorizr

### ISSUE THREAD

**Issue by sathya4code** [ueman/drawable#3]
Hi,

I am using flavours in my app. I want to display different image using flavour. For that I am using Flutter Flavorizr.

When I am running project in debug mode, image is showing from res folder. But when I am running release mode, image not showing. and didn't get any exception.

Command to run debug mode:
`flutter run --flavor ascham -t lib/main-ascham.dart`

Command to run release mode:
`flutter run --release --flavor ascham -t lib/main-ascham.dart`

I have attached the sample project for your reference. Please let me know for further clarification if required.

**Note: App launcher icon changing based on the flavor.** 

Thanks,
Sathya.

[flutter_flavors.zip](https://github.com/ueman/drawable/files/6911393/flutter_flavors.zip)

**Comment by sathya4code** [comment]
Hi,

Any update?

**Comment by ueman** [comment]
No, not yet. I'm currently really constraint on time. Hoewever I will accept PRs, so you could fix it yourself and I'll publish an update.

**Comment by nirranthea** [comment]
Hi, I have same issue.

I am using Flavor too, but not with Flutter Flavorizr. I am disabling `shrinkResources` & `minifyEnabled` on `build.gradle` for the meantime, so `flutter run --release --flavor ...` worked with images showing.
